packages/atlas/src/api/client/index.ts

```diff
@@ -109,6 +109,12 @@ const createApolloClient = () => {
               logDistributorPerformance(resolvedUrl, eventEntry)
               return resolvedUrl
             } catch (err) {
+              if (err instanceof MediaError) {
+                SentryLogger.error('Error during asset download test, media is not supported', 'AssetsManager', err, {
```

Comment:
There was logging directly by the plugin before that the media type isn't supported. This change adds a rather generic console error. Without checking the log common users still have no clue what is wrong and where to look , the video just won't play without specific error message.

Reply:
I don't think that's correct.  [This change](https://github.com/Joystream/atlas/pull/4149/files#diff-7cd3fa81e7c8a9694c1b2265bb611db21ee1cec0f4e6d53336a97e5d1be482f8R296) will make sure users will see the screen error with a message: 
```
The video could not be played because the data is corrupted or the encoding is not supported. This issue has been logged. If you need support, reach out to our community on Discord.
```
I think that's enough.